### PR TITLE
fix: remove DropdownButton animation and add offset props

### PR DESCRIPTION
**Pull request by sylvanasantos**
### Overview
As per updated DropdownButton designs:

Screen Shot 2021-05-17 at 5 58 29 PM

- removed the open/close animation
- added props for offsetting the positioning of the DropdownList

### PR Checklist

- [ ] Related to designs:
- [ ] Related to JIRA ticket: ABC-123
- [ ] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

File: packages/gamut-styles/utils/variables/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,189 @@
+export namespace colors {
```
Comment (codecaaron) [inline packages/gamut-styles/utils/variables/index.d.ts:1]: This got accidentally generated from a build probably

**Review by codecaaron — CHANGES_REQUESTED** [review]
LGTM, just remove the erroneous definition file :)

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60a548a1a8b35522b1a8f8cd--gamut-preview.netlify.app>https://60a548a1a8b35522b1a8f8cd--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60a548a1a8b35522b1a8f8cd>Deploy Logs</a></p>

**Review by codecaaron — APPROVED** [review]

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.106-alpha.d3df97.0<br/>@codecademy/gamut-labs@17.3.1-alpha.d3df97.0<br/>@codecademy/styleguide@38.3.1-alpha.d3df97.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60a54a692f0ac3239b9fea46--gamut-preview.netlify.app>https://60a54a692f0ac3239b9fea46--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60a54a692f0ac3239b9fea46>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.107-alpha.f8cc0a.0<br/>@codecademy/gamut-labs@17.3.2-alpha.f8cc0a.0<br/>@codecademy/styleguide@38.3.2-alpha.f8cc0a.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60a55d80e6ce0b33b6874410--gamut-preview.netlify.app>https://60a55d80e6ce0b33b6874410--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60a55d80e6ce0b33b6874410>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.107-alpha.8ee500.0<br/>@codecademy/gamut-labs@18.0.0-alpha.8ee500.0<br/>@codecademy/styleguide@39.0.0-alpha.8ee500.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60a56149e33f0d36f3e56c4e--gamut-preview.netlify.app>https://60a56149e33f0d36f3e56c4e--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60a56149e33f0d36f3e56c4e>Deploy Logs</a></p>

**Pull request merged**